web/pgadmin/static/js/Theme/index.jsx

```diff
@@ -774,7 +774,7 @@ function getFinalTheme(baseTheme) {
 export default function Theme({children}) {
   const prefStore = usePreferences();
   const [theme, setTheme] = useState(prefStore.getPreferencesForModule('misc')?.theme);
-  const [selectedMode, setSelectedMode] = useState(prefStore.getPreferencesForModule('misc')?.theme || 'light');
+  const currentTheme = usePreferences().getPreferencesForModule('misc').theme;
```

Comment:
This can be moved inside useEffect with `currentTheme  = prefStore.getPreferencesForModule('misc').theme`

Reply:
Done.